projects/packages/explat/src/client/assignment.ts

```diff
@@ -0,0 +1,19 @@
+import apiFetch from '@wordpress/api-fetch';
+import { addQueryArgs } from '@wordpress/url';
+
+export const fetchExperimentAssignment = async ( {
+	experimentName,
+	anonId,
+}: {
+	experimentName: string;
+	anonId: string | null;
+} ): Promise< unknown > => {
+	const params = {
+		experiment_name: experimentName,
+		anon_id: anonId ?? undefined,
+		as_connected_user: false,
```

Comment:
There are some options for exposing this in the explat client API if you'd like to do that.

Reply:
Oh, that's cool. I can work on a follow-up later